lib/Extension/LanguageServerCodeTransform/CodeAction/ExtractConstantProvider.php

```diff
@@ -4,6 +4,7 @@
 
 use Amp\CancellationToken;
 use Amp\Promise;
+use LanguageServerProtocol\CodeActionKind;
```

Comment:
this is the "wrong" the one phpactor uses is `phpactor/language-server-protocol` this one is an incidental dependency from Psalm.

AFAIK the code action "kinds" are not standardised and that class is perhaps specific to the felixbecker language server. We couild perhaps add a similar class in `lib/Extension/LanguageServer` if it adds benefit.